ai-ml/maxdiffusion-tpu/build/server/requirements.txt

```diff
@@ -1,8 +1,8 @@
 uvicorn[standard]==0.30.6
 fastapi[all]==0.114.0
 pillow
-jax==0.4.31
-jaxlib==0.4.31
+#jax==0.4.31 # installed in Dockerfile
+#jaxlib==0.4.31 # installed in Dockerfile
```

Comment:
**Question:** Is there a reason we now install these via the Dockerfile?

Reply:
@NimJay no, I've moved it into the requirements file with latest commit

Reply:
@NimJay please merge with main. Thank you